Create Readme.md

nicole-w-0 opened the pull request:
## Submission Checklist:

<!--To check the checkboxes, simply replace the empty space in the brackets with a captial X, like this: [ ] becomes [X]-->

- [X] I am a current high school, middle school, or home schooled student.
  - [X ] I am 18 or under
  - [X ] I have filled out the [verification form](https://verify.onboard.hackclub.com) **(do this for every pull request)**
- [ ] I have joined the [`#electronics`](https://hackclub.slack.com/archives/C056AMWSFKJ) channel on [Slack](https://hackclub.com/slack/?event=onboard)
- [X ] I made this board from scratch, even if I followed a tutorial
- [X ] I have followed the directions in [README.md](https://github.com/hackclub/OnBoard/blob/main/README.md#getting-started)
  - [ X] Created a folder under `onboard/projects`
  - [X ] Filled out `TEMPLATE.md` as `README.md` inside your project folder
  - [x] Uploaded `gerber.zip` and `schematic.pdf` inside your project folder
  - [x] Created a folder called `src` and uploaded design files
  - [x] Uploaded screenshot of PCB order called `cart.png` with all costs included
  - [X ] I am ordering the smallest batch size (ie. JLCPCB's smallest order is for 5 boards)
  - [x] If outside the US I've checked that I can afford the customs charges in my country (which isn't covered by the grant)
___
- [ ] *(Optional)* This project is from a tutorial[^1].
- [ ] *(Optional)* This project is for OnBoard Live[^2].
- [ ] *(Optional)* I'm in a FIRST (FRC, FTC, FLL, etc.) team. The number is: ____

[^1]: Projects from a tutorial are 100% fine! We just want to ask so we can count how people are using tutorials.
[^2]: OnBoard Live is a special version of OnBoard where you can earn more money for designing advanced boards. Check out the #onboard-live channel on [our Slack](https://hackclub.com/slack/?event=onboard)!
<!-- -Submission- -->

prophetorpheus commented:
<!-- MY-ONBOARD-BOT --> Hi, I'm Orpheus Leap! Here to help you review your PR.
  
**projects/Dragon_board:**

Required files
||||
|---|---|---|
|✅| README.md | A description of your project |
|❌ | cart.png | You need to include a screenshot of your JLCPCB. Check out [these instructions](https://github.com/hackclub/OnBoard/blob/main/docs/ordering_from_JLCPCB.md#pcb-review). If you already have one, make sure it's a PNG file named exactly "cart.png". |

**❌❌❌Cannot find a Gerber File. Please export a gerber from your PCB Design software and name it exactly "gerber.zip".**
  Happy OnBoarding!
 <sub>This bot is a simpler helper for common submission types. If there's an error with this, please point it out and someone from the OnBoard team will look at it manually.</sub>

maxwofford commented:
Hey @nicole-w-0, looks like you just uploaded a text file here instead of a board design. Do you need help getting the full thing submitted?

nicole-w-0 commented:
Hey Max, the files should have just been uploaded, sorry about that. I'm unable to attach the cart file right now, as there are parts that I want but are out of stock. I'm working with a mentor to solve the issue.

nicole-w-0 commented:
whoops

maxwofford commented:
@nicole-w-0 no sweat, you've got this! just lmk if you need any help getting those files in. If it helps there's a short tutorial i made here: https://github.com/hackclub/OnBoard?tab=readme-ov-file#getting-started

nicole-w-0 commented:
Hey Max, are all files uploaded?

maxwofford commented:
Nice work on this! I just sent your grant card https://hcb.hackclub.com/grants/nZI4kK

nicole-w-0 commented:
Hey Max,

I received a grant of $50, but the total cost is $83.52 (see attached
file). Could you please send the $33.52? Thank you!

Best,

Nicole

On Thu, Apr 3, 2025 at 12:07 PM Max Wofford ***@***.***>
wrote:

> Merged #1553 <https://github.com/hackclub/OnBoard/pull/1553> into main.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hackclub/OnBoard/pull/1553#event-17119203540>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BQKOZA7SXYV4CPB5UXKWTNT2XWBHTAVCNFSM6AAAAABZQYP6LWVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJXGEYTSMRQGM2TIMA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

nicole-w-0 commented:
Thank you!

On Sun, Apr 6, 2025 at 7:21 PM Asher Edwards ***@***.***>
wrote:

> Hey @nicole-w-0 <https://github.com/nicole-w-0>, I just took care of this
> for you since max is usually busy. I added 50 dollars to your existing
> grant, you should be good to go!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hackclub/OnBoard/pull/1553#issuecomment-2781863818>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BQKOZAYEYOS7DNVBLJQREOD2YHOL3AVCNFSM6AAAAABZQYP6LWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOBRHA3DGOBRHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: techy-robot]*techy-robot* left a comment (hackclub/OnBoard#1553)
> <https://github.com/hackclub/OnBoard/pull/1553#issuecomment-2781863818>
>
> Hey @nicole-w-0 <https://github.com/nicole-w-0>, I just took care of this
> for you since max is usually busy. I added 50 dollars to your existing
> grant, you should be good to go!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/hackclub/OnBoard/pull/1553#issuecomment-2781863818>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BQKOZAYEYOS7DNVBLJQREOD2YHOL3AVCNFSM6AAAAABZQYP6LWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOBRHA3DGOBRHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
